cdc/processor/sinkmanager/table_sink_wrapper.go

```diff
@@ -129,6 +132,7 @@ func (t *tableSinkWrapper) start(startTs model.Ts, replicateTs model.Ts) {
 	// Because in two phase scheduling, the table sink may be advanced to a later ts.
 	// And we can just continue to replicate the table sink from the new start ts.
 	t.checkpointTs.Store(startTs)
+	t.receivedSorterResolvedTs.Store(startTs)
```

Comment:
Do we need to add a check here? like if receivedSorterResolvedTs less than startTs.